http-client-spi/src/main/java/software/amazon/awssdk/http/DefaultSdkHttpFullRequest.java

```diff
@@ -97,6 +97,14 @@ private Integer standardizePort(Integer port) {
         return port;
     }
 
+    private ContentStreamProvider standardizeBody(ContentStreamProvider contentStreamProvider) {
+        if (httpMethod.equals(SdkHttpMethod.GET)) {
+            return null;
```

Comment:
Should we warn or throw an exception if there was some body that got ignored because of the method?